interp/trace.go

```diff
@@ -14,7 +14,7 @@ import (
 type tracer struct {
 	buf       bytes.Buffer
 	printer   *syntax.Printer
-	stdout    io.Writer
+	stderr    io.Writer
```

Comment:
nit: might as well call this `output`, because it has just one output writer, and in the future we'll likely need to support outputs other than stderr for better bash compatibility

Reply:
done.